src/lib/css-parse.html

```diff
@@ -89,6 +90,21 @@
       return node;
     },
 
+    // conversion of sort unicode escapes with spaces like `\33 ` (and longer) into
+    // expanded form that doesn't require space `\000033`
+    _expandUnicodeEscapes : function(s) {
+      return s.replace(/\\([0-9a-f]{1,6})(\s?)/gi, function(source, code, whitespace) {

```

Comment:
I don't think we need to modify escape sequences _unless_ they have a trailing space.